src/Nox.Solution/Models/Infrastructure/Endpoints/Endpoints.cs

```diff
@@ -11,20 +11,20 @@ public class Endpoints
     {
         [Title("The api route prefix, defaults to api/v1 or to api/vMajor({Solution.Version}) if Version is set in the root of the Solution.")]
         [Description(@"Defines the prefix for all Api routes end points.")]
-        public string ApiRootPrefix { get; internal set; } = null!;
+        public string ApiRoutePrefix { get; internal set; } = null!;
         public ApiServer? ApiServer { get; internal set; }
         public BffServer? BffServer { get; internal set; }
                 
         public void ApplyDefaults(string solutionVersion)
         {
-            if (ApiRootPrefix is null || string.IsNullOrEmpty(ApiRootPrefix))
+            if (ApiRoutePrefix is null || string.IsNullOrEmpty(ApiRoutePrefix))
```

Comment:
```suggestion
            if (string.IsNullOrEmpty(ApiRoutePrefix))
```

You don't need to check if it is null, because IsNullOrEmpty should already be doing that.